pyms/config/conf.py

```diff
@@ -144,3 +146,61 @@ def __verify_deprecated_env_variables(config):
         except AttrDoesNotExistException:
             pass
         logger.warning(msg)
+
+
+def create_conf_file(use_requests: bool = False, use_swagger: bool = False) -> Union[Exception, str]:  # pragma: no cover
```

Comment:
remove `# pragma: no cover` and add tests

Reply:
Added test for create_conf_file